src/org/jitsi/impl/neomedia/rtp/translator/SimulcastController.java

```diff
@@ -0,0 +1,532 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.impl.neomedia.rtp.translator;
+
+import net.sf.fmj.media.rtp.*;
+import org.jitsi.impl.neomedia.*;
+import org.jitsi.impl.neomedia.rtcp.*;
+import org.jitsi.impl.neomedia.rtp.*;
+import org.jitsi.impl.neomedia.transform.*;
+import org.jitsi.service.neomedia.*;
+import org.jitsi.util.*;
+
+import java.lang.ref.*;
+
+/**
+ * Filters the packets of
+ * {@link MediaStreamTrackImpl} based on the currently forwarded subjective
+ * quality index. It's also taking care of upscaling and downscaling. As a
+ * {@link PacketTransformer}, it rewrites the forwarded packets so that the
+ * gaps as a result of the drops are hidden.
+ *
+ * @author George Politis
+ */
+public class SimulcastController
+    implements TransformEngine
+{
+    /**
+     * The {@link Logger} to be used by this instance to print debug
+     * information.
+     */
+    private final Logger logger = Logger.getLogger(SimulcastController.class);
+
+    private static final SimTransformation dropState
```

Comment:
I think capital casing is reserved for primitive type constants only. For example, the logger that we use in every class is a constant, but it's not in capital case.